src/foam/nanos/notification/services

```diff
@@ -21,7 +21,7 @@ p({
         new foam.nanos.notification.SendSlackNotificationDAO(x)))).build())
       .setJournalType(foam.dao.JournalType.SINGLE_JOURNAL)
       .setJournalName("notifications")
-      .setInnerDAO(dao)
+      .setMDAO(dao)
```

Comment:
Just replace the index setup with 
setIndex( new foam.core.PropertyInfo[] {
...
}
on the EasyDAO, and don't explicitly set the MDAO.

Reply:
I don’t think setIndex can set multiple index

Reply:
yes it can.  it takes a PropertyInfo[]